students/15th/kimbyulyi/styles/login.css

```
@@ -0,0 +1,179 @@
+@import "reset.css";
+@import "common.css";
+
+/* Main */
+
+main {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-items: center;
+}
+
+/* Article  */
+
+article {
+  display: flex;
+  max-width: 935px;
+  margin-top: 110px;
+}
+
+/*right section*/
+
+.login__container {
+  display: flex;
+  flex-direction: column;
+  width: 351px;
+  height: 618px;
+  padding: 10px 0px;
+  background-color: var(--color-white);
+}
+
+/* 로그인 컨테이너 상단 */
+
+.login__container__top {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 20px;
+  border: 1px solid var(--color-gainsoboro);
+}
+
+.login__form {
+  display: flex;
+  flex-direction: column;
+  margin: 30px 0px;
+}
+
+.login__form input {
+  width: 265px;
+  height: 25px;
+  padding: 15px 5px;
+  margin-bottom: 10px;
+  background-color: var(--color-snow);
+  border: 1px solid var(--color-gainsoboro);
+  border-radius: 3px;
+  font-size: var(--font-small);
+  color: var(--color-nobel);
+}
+
+.loginBtn {
+  padding: 8px 0px;
+  border-radius: 5px;
+  font-weight: var(--weight-regular);
+  color: var(--color-white);
+  background-color: var(--color-blue);
+  cursor: pointer;
+}
+
+.loginBtn:disabled {
+  opacity: 0.4;
+  cursor: default;
+}
+
+.login__bar {
+  display: flex;
+  width: 268px;
+  height: 15px;
+  justify-content: center;
+  align-items: center;
```

연관된 속성끼리는 가독성을 위해 인접하게 작성해주세요! 
```suggestion
  display: flex;
  justify-content: center;
  align-items: center;
  width: 268px;
  height: 15px;
```

---

넵 참고하겠습니다!✅